projects/admin/src/app/record/detail-view/acquisition-order-detail-view/order-lines/acquisition-order-lines.component.ts

```diff
@@ -14,54 +14,53 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { RecordService, RecordUiService } from '@rero/ng-core';
+import { RecordPermissionService } from 'projects/admin/src/app/service/record-permission.service';
 import { Observable } from 'rxjs';
-import { RecordPermissionMessageService } from '../../../../service/record-permission-message.service';
 
 @Component({
   selector: 'admin-acquisition-order-lines',
   templateUrl: './acquisition-order-lines.component.html'
 })
-export class AcquisitionOrderLinesComponent {
+export class AcquisitionOrderLinesComponent implements OnInit {
 
   /** Acquisition order pid */
   @Input() order: any;
 
   /** Event for delete order line */
-  @Output()
-  deleteOrderLine = new EventEmitter();
+  @Output() deleteOrderLineEmitter = new EventEmitter();
```

Comment:
If we follow the not-yet-validated guidelines, you should put `@output` just before constructor

Reply:
done